xonsh/aliases.py

```diff
@@ -73,6 +73,19 @@ def source_bash(args, stdin=None):
     return
 
 
+def exec(args, stdin=None):
```

Comment:
If a new alias was added would it still solve #84, or would users just need to manage that themselves (e.g. exec aliased to xexec in xonshrc)